shared/javascripts/network_service.js

```diff
@@ -433,20 +433,24 @@ var privlyNetworkService = {
     privlyNetworkService.mobileHide();
   },
 
-  /**
-   * Show/hide the appropriate navigation items for when the user is logged out.
-   */
-  showLoggedOutNav: function() {
-    
-    // Don't show the nav at all if the content is injected.
-    if(typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
-      return;
+ /**
+ * Show/hide the appropriate navigation items for when the user is logged out.
+ */
+showLoggedOutNav: function() {
```

Comment:
You are still improperly formatting the whole block. See how you changed existing lines with only whitespace changes?